app/controllers/concerns/foreman/controller/registration_commands.rb

```diff
@@ -18,7 +18,7 @@ def registration_args
   end
 
   def insecure
-    registration_params['insecure'] ? ' --insecure' : ''
+    registration_params['insecure'] ? ' --insecure ' : ' '
```

Comment:
Why don't we just add the space in the command as before? I don't think it belongs here.